src/styles/index.scss

```diff
@@ -1,3 +1,78 @@
-body {
-  margin: 0;
+@use 'sass:list';
+@import 'variables';
```

Comment:
The `@import` rule is used here, which is generally discouraged in favor of `@use` or `@forward` in modern Sass. Consider replacing `@import 'variables';` with `@use 'variables';` to follow the latest Sass guidelines.